src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html

```diff
@@ -300,12 +300,63 @@ <h3>Backup Source Configuration</h3>
             </div>
         </div>
 
-        <div class="form-group row" data-bind="validationElement: backupLink, visible: source() === 'cloud'">
-            <div class="col-md-4">
-                <label class="control-label">Backup Link</label>
+        <div data-bind="visible: source() === 'cloud'">
+            <div class="form-group row" data-bind="validationElement: backupLink">
+                <div class="col-md-4">
+                    <label class="control-label">Backup Link</label>
+                </div>
+                <div class="col-md-8">
+                     <input class="form-control" data-bind="textInput: backupLink" placeholder="Enter backup link from RavenDB Cloud" />
+                </div>
             </div>
-            <div class="col-md-8">
-                 <input class="form-control" data-bind="textInput: backupLink" />
+            <div data-bind="if: $root.databaseModel.spinners.backupCredentialsLoading()">
+                <div class="form-group row">
+                    <div class="col-md-offset-4 col-md-8">
+                        <label class="control-label global-spinner spinner-sm"></label>
+                    </div>
+                </div>
+            </div>
+            <div data-bind="if: !$root.databaseModel.spinners.backupCredentialsLoading() && cloudCredentials() && backupLink.isValid()">
+                <div class="form-group row">
+                    <div class="col-md-4">
+                        <label class="control-label">Backup Storage Type</label>
+                    </div>
+                    <div class="col-md-8">
+                        <label class="control-label text-info" data-bind="text: cloudCredentials().BackupStorageType"></label>
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <div class="col-md-4">
+                        <label class="control-label">Aws Region Name</label>
```

Comment:
Remove 'Aws' user will know that from 'Backup Storage Type', right?

Reply:
done